### PR TITLE
[FEATURE] subscription throttling using ngsi v2

### PR DESCRIPTION
**Pull request by iariasleon**
```
                    SUMMARY:
-------------------------------------------------
  - components\ngsiv2\subscriptions\create_a_new_subscription\create_a_new_subscription_throttling.feature >> passed: 16, failed: 0, skipped: 0 and total: 16 with duration: 4.012 seconds.
-------------------------------------------------
  Date: Monday, June 20, 2016 14:19:59
-------------------------------------------------
1 feature passed, 0 failed, 0 skipped
16 scenarios passed, 0 failed, 0 skipped
89 steps passed, 0 failed, 0 skipped, 0 undefined
Took 0m4.012s
```
@kzangeli @fgalan @crbrox

### CODE REVIEW

File: test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      | parameter   | value                    |
+      | error       | BadRequest               |
+      | description | throttling is not an int |
+    Examples:
+      | throttling    |
+      | true          |
+      | ["er", 34]    |
+      | {"value": 34} |
+      | "234324324"   |
+      | 23432.45      |
```
Comment (kzangeli) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature]: Does a float provoke an error for throttling?
I thought we might 'round away' those decimals, but give an error ... Doesn't seem 100% right to me.

File: test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    When create a new subscription in raw mode
+    Then verify that receive a "Bad Request" http code
+    And verify an error response
+      | parameter   | value                                |
+      | error       | ParseError                           |
+      | description | Errors found in incoming JSON buffer |
+    Examples:
+      | throttling |
+      | 23432,45   |
+      | (23432,45) |
```
Comment (kzangeli) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature]: What's the difference between the last two tests?
'Not int type' and 'invalid value' for throttling ...
To me it seems they should both give the same error

File: test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature]: now, a decimal returns a "throttling is not an int" error

File: test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature]: 23432**,**45 != 23432**.**45
the first use a comma (invalid in json) and the second a dot (not allowed in throttling, but valid in json)

Comment (kzangeli) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature]: Yes, I saw that but I didn't reflect upon it. These are not strings and give a json parse error. OK

**Comment by fgalan** [comment]
LGTM

File: test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature (hunk 2 above)
Comment (kzangeli) [inline test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_throttling.feature]: NTC

**Pull request merged**